Lib/pstats.py

```diff
@@ -25,9 +25,28 @@
 import time
 import marshal
 import re
+from enum import Enum
 from functools import cmp_to_key
 
-__all__ = ["Stats"]
+__all__ = ["Stats", "SortKey"]
+
+
+class SortKey(str, Enum):
+    CALLS = 'calls'
+    NCALLS = 'ncalls'
+    CUMULATIVE = 'cumulative'
+    CUMTIME = 'cumulative'
+    MODULE = 'module'
+    FILE = 'file'
+    FILENAME = 'filename'
```

Comment:
MODULE, FILE, and FILENAME should all map to the same constant (I would choose either "module" or "filename").